src/app-styles.module.scss

```diff
@@ -14,4 +14,5 @@
 
 .mobile {
   height: 100svh;
+  background-image: url('./assets/images/landing-bg.svg');
```

Comment:
Why this?

Reply:
The background of the scene was missing in the landing but we don't have a scene in mobile so I put it in png.